Automattic/wp-calypso
# Sort Sites Dashboard by Last Publish

#### Proposed Changes

We've already added the Last Publish column that uses the site's `updated_at` field to display nicely formatted dates.

In this PR I've added a hook that sorts sites using the Last Publish field.

#### Testing Instructions



1. Load Sites Dashboard at `/sites-dashboard` URL
2. Confirm that sites are sorted by Last Publish date, descending - the latest updated site should be the first one.
3. Confirm that sorting works when we switch tabs and when we filter the list using search phrase.

#### Pre-merge Checklist






Related to https://github.com/Automattic/wp-calypso/issues/65168
